File: sdkwork-app-sdk-go/types/page_sku_vo.go
package types


type PageSkuVO struct {
	TotalPages int `json:"totalPages"`
	TotalElements int `json:"totalElements"`
	Size int `json:"size"`
	Content []SkuVO `json:"content"`
	Number int `json:"number"`
	Sort SortObject `json:"sort"`
	First bool `json:"first"`
	Last bool `json:"last"`
	NumberOfElements int `json:"numberOfElements"`
	Pageable PageableObject `json:"pageable"`
	Empty bool `json:"empty"`
}

File: sdkwork-app-sdk-typescript/src/types/page-sku-vo.ts
import type { PageableObject } from './pageable-object';
import type { SkuVO } from './sku-vo';
import type { SortObject } from './sort-object';

export interface PageSkuVO {
  totalElements?: number;
  totalPages?: number;
  size?: number;
  content?: SkuVO[];
  number?: number;
  first?: boolean;
  last?: boolean;
  numberOfElements?: number;
  sort?: SortObject;
  pageable?: PageableObject;
  empty?: boolean;
}
